### ISSUE TITLE
List of maps already played

### ISSUE THREAD

**Issue by AnSch1510** [MaSch0212/minigolf-friday#122]
There should be a list of maps which have been already played. So we have track of what map is available to play in the future

**Comment by MaSch0212** [comment]
You probably want this per timeslot, right? Might be difficult to define, das the events are created manually.

**Comment by AnSch1510** [comment]
Yes, per timeslot. Maybe a event view which merges the timeslot logs together ordered by time

**Comment by MaSch0212** [comment]
Maybe we need to combine this with #10.

**Comment by AnSch1510** [comment]
Isn't it possible to add en entry to a new table each time a registration/deregistration has heapened and add the timeslot id to it

**Comment by AnSch1510** [comment]
Damn I had mixed this issue with the log-issue 🫣

**Comment by AnSch1510** [comment]
> Maybe we need to combine this with #10.

Would be handy

Then we would have a dedicated id we can have track of
